pack/CoS.json

```diff
@@ -0,0 +1,166 @@
+        "traits": "",
+        "type_code": "event"
+    },
+    {
+        "code": "13027",
+        "cost": "4",
+        "deck_limit": 3,
+        "faction_code": "baratheon",
+        "flavor": "",
+        "illustrator": "Adam Lane",
+        "is_intrigue": false,
+        "is_loyal": false,
+        "is_military": true,
+        "is_multiple": false,
+        "is_power": true,
+        "is_unique": false,
+        "name": "Red Keep Gaoler",
+        "octgn_id": null,
+        "pack_code": "CoS",
+        "position": 27,
+        "quantity": 3,
+        "strength": 4,
+        "text": "Shadow (4).\n<b>Action:</b> Move 2 power from your faction card to a [baratheon] character to put Red Keep Gaoler into play from shadows.",
+        "traits": "Ally. Guard.",
+        "type_code": "character"
+    },
+    {
+        "code": "13029",
+        "cost": "4",
+        "deck_limit": 3,
+        "faction_code": "lannister",
+        "flavor": "",
+        "illustrator": "Antonio José Manzanedo",
+        "is_intrigue": true,
+        "is_loyal": false,
+        "is_military": true,
+        "is_multiple": false,
+        "is_power": false,
+        "is_unique": false,
+        "name": "The Regent's Guard",
+        "octgn_id": null,
+        "pack_code": "CoS",
+        "position": 29,
+        "quantity": 3,
+        "strength": 4,
+        "text": "Shadow (4).\n<b>Action:</b> Return a [lannister] non-<i>Ally</i> character with printed cost 4 or higher to your hand to put The Regent's Guard into play from shadows.",
+        "traits": "Ally. Guard.",
+        "type_code": "character"
+    },
+    {
+        "code": "13034",
+        "cost": "-",
+        "deck_limit": 3,
+        "faction_code": "targaryen",
+        "flavor": "",
+        "illustrator": "Paolo Puggioni",
+        "is_loyal": true,
+        "is_multiple": false,
+        "is_unique": false,
+        "name": "Shadow of the East",
+        "octgn_id": null,
+        "pack_code": "CoS",
+        "position": 34,
+        "quantity": 3,
+        "text": "Shadow (0).\n<b>Choose an attachment and discard it from play. If there is a <i>Summer</i> plot card revealed, return Shadow of the East to your hand instead of placing it in your discard pile.",
+        "traits": "",
+        "type_code": "event"
+    },
+    {
+        "code": "13036",
+        "cost": "-",
+        "deck_limit": 3,
+        "faction_code": "martell",
+        "flavor": "",
+        "illustrator": "Logan Feliciano",
+        "is_loyal": true,
+        "is_multiple": false,
+        "is_unique": false,
+        "name": "Shadow of the South",
+        "octgn_id": null,
+        "pack_code": "CoS",
+        "position": 36,
+        "quantity": 3,
+        "text": "Shadow (1).\n<b>Choose a character. Until the end of the phase, that character loses a challenge icon of your choice. If there is a <i>Scheme</i> plot card revealed, return Shadow of the South to your hand instead of placing it in your discard pile.",
```

Comment:
@ystros fyi- the `<b>` tag needs to be closed. couldn't find it in the upstream repo, fixing it here now.